app/lib/package/backend.dart

```diff
@@ -1268,7 +1268,8 @@ class PackageBackend {
 
     // TODO: consider allowing other events from
     //       https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
-    if (agent.payload.eventName != 'push') {
+    if (agent.payload.eventName != 'push' &&
+        agent.payload.eventName != 'workflow_dispatch') {
       throw AuthorizationException.githubActionIssue(
           'publishing is only allowed from "push" events, this token originates from a "${agent.payload.eventName}" event');
```

Comment:
We probably wanna adjust this error message

Reply:
Good catch, updated!